spec/defines/dotfile_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper'
+
+describe 'python::dotfile', type: :define do # rubocop:disable RSpec/MultipleDescribes
+
+  context 'on Debian OS' do
+    let :facts do
```

Comment:
can you please use [rspec-puppet-facts](https://github.com/mcanevet/rspec-puppet-facts) for this?

Reply:
Yes.